ui/src/containers/VolumeInformation.js

```diff
@@ -60,14 +66,23 @@ const InformationValueSection = styled.div`
   min-width: 500px;
 `;
 
+const LoaderContainer = styled(Loader)`
+  position: absolute;
```

Comment:
Avoid `position: absolute;`  if you can please

Reply:
Ok. I removed it. 
Using `visibility` to control the visibility of the `<Loader>`